corehq/apps/users/views/mobile/groups.py

```diff
@@ -201,5 +202,11 @@ def page_context(self):
             'group': self.group,
             'bulk_sms_verification_enabled': bulk_sms_verification_enabled,
             'num_users': len(self.member_ids),
-            'user_form': self.user_selection_form
+            'user_form': self.user_selection_form,
+            'domain_uses_case_sharing': self.domain_uses_case_sharing,
         }
+
+    @property
+    def domain_uses_case_sharing(self):
+        domain = Domain.get_by_name(Group.get(self.group_id).domain)
+        return domain.case_sharing_included()
```

Comment:
This iterates though every app in the domain with no caching, probably going to be slow (domain.case_sharing doesn't seem to get set anywhere).